cypress/integration/edit_user_profile_spec.js

```diff
@@ -0,0 +1,69 @@
+/// <reference types="Cypress" />
+import { aliasQuery, aliasMutation } from '../../src/utilities/graphql-test-utils';
+
+describe("Edit User Profile Page Spec", () => {
+
+    beforeEach(() => {
+        cy.intercept('POST', 'https://intense-ocean-61260.herokuapp.com/graphql', (req) => {
```

Comment:
Thank you for figuring out how to get Cypress stubbed with graphql, this is great!